app/src/main/res/drawable/record_icon.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0"
+        android:tint="?attr/colorControlNormal">
```

Comment:
Applying tint attributes shows a warning
> Resource references will not work correctly in images generated for this vector icon for API < 21

Current min SDK is 16. Please remove this tint attribute from icons in this PR.

Reply:
Sure @CloudyPadmal